hax-types/src/cli_options/mod.rs

```diff
@@ -215,12 +215,25 @@ enum InclusionKind {
 
 #[derive_group(Serializers)]
 #[derive(JsonSchema, Debug, Clone)]
-struct InclusionClause {
-    kind: InclusionKind,
-    namespace: Namespace,
+pub struct InclusionClause {
+    pub kind: InclusionKind,
+    pub namespace: Namespace,
 }
 
-fn parse_inclusion_clause(
+impl ToString for InclusionClause {
+    fn to_string(&self) -> String {
+        let kind = match self.kind {
+            InclusionKind::Included(DepsKind::Transitive) => "+",
```

Comment:
It's a detail but it might be better to have `"+~"`, `"+:"` and the correspondance with the different cases for `InclusionKind` in a method or in a global variable that would be used both here and in `parse_inclusion_clause`. Just to avoid that somehow they end up diverging in some way.

Reply:
That's a great idea, thanks, I pushed a fix! :)